doc/mvf_v4.rst

```diff
@@ -292,9 +313,9 @@ Some common points to note that about the solutions:
   all the data was flagged).
 
 - Solutions are only valid as long as the system gain controls (TODO:
-  name for these) are not altered. It is thus not generally advisable to
-  re-use gains from one capture block to correct data from another
-  capture block.
+  name for these) are not altered. Re-using gains from one capture block to
+  correct data from another capture block may yield inocrrect results
```

Comment:
incorrect

Reply:
Yes, it was incorrectly spelt :-) Fixed.